### ISSUE TITLE
Add addresses in Muskogee, OK

### ISSUE THREAD

**Issue by nvkelso** [openaddresses/openaddresses#2004]
Status:
- [ ] **Research sources** and propose best source (and it's data license)
- [ ] **Review proposed source**
- [ ] **Approve source** based on review comments
- [ ] **Add** to OpenAddresses

_NOTE: Proposed source notes should include basic column mapping to OpenAddress data model._

**Comment by justinelliotmeyers** [comment]
anyoneone know how to convert autocad to reality!? http://www.muskogeeonline.org/departments/planning/city_maps.php#revize_document_center_rz269

e-mailed asking for gis data ;)
jzwirtz@muskogeeonline.org

**Comment by nvkelso** [comment]
Eek, that is some crazy!

**Comment by justinelliotmeyers** [comment]
yes - just got off the phone w the city. county only has in cad... I'm calling to confirm this. the city has data and will be sending to us (*heavily outdated), but the guy said he would never convert the data from cad to gis ever again!?! stay tuned...

**Comment by justinelliotmeyers** [comment]
Sarah with the county assessor's office will be back tomorrow and should call me.

**Comment by justinelliotmeyers** [comment]
*uploading to dropbox now and reaching out to e911:

Justin,

Here is the address shapefile, the latest updates I believe to this was in 2014.  I will send what parcels I have in another email, I was exceeding the email attachment limit size with both.   If you need the addresses for entire county maybe Vicki over at 911 might be able to help you out.

vatchley@mcc911.org


Jimmy

**Comment by justinelliotmeyers** [comment]
just e-mailed Vicki - city data here: https://www.dropbox.com/sh/vlybcs3sjt5nw3b/AADRst1-LEOmXVZ5bicfMFWTa?dl=0

**Comment by justinelliotmeyers** [comment]
Sorry Justin, I thought I had sent everything for that shapefile.  Here is that dbf file.  I tried sending that parcel but I got a return email.   IT said it looked like you guy’s server denied the email  because the attachment size was too large 18.7MB. That was just the dbf file alone.   IT said we could do a FTP file transfer for the parcel shapefile and also for this, if this dbf file doesn’t go through.

I should mention for this address shapefile, I’m not sure what it is, I made a GIS ID for making “Joins” and it seems to work good as a shapefile, but when it’s in a geodatabase, some of the records don’t seem to join correctly.

Jimmy
